### DISCUSSION TITLE
How do I actually implement Podcast streaming?

### DISCUSSION THREAD

**Discussion by PassiveLemon** [BLeeEZ/amperfy#222] [Q&A] [answered]
I can't find good information on how to implement it. I already stream music with Navidrome, which works great, and would like to stream my downloaded podcasts too. I'm assuming this is something dependent on my music streamer but I'm not sure if there's certain criteria that need to be met to allow this.

**Comment by BLeeEZ** [comment, marked as answer]
Hi,
you need a server that supports podcasts like https://ampache.org or https://airsonic.github.io
Then add there the podcasts you want to listen to and Amperfy can sync the info to your iOS device.

**Reply by PassiveLemon** [reply]
Got it, thanks
